### PR TITLE
HCAP-1191 | Security Alert Fix | Add validation and sanitization on user input

### PR DESCRIPTION
**Pull request by fwpushan**
- [x] Add sanitization on user input
- [x] Update error messages.

### CODE REVIEW

File: server/routes/participant-user.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ router.patch(
       });
       res.status(200).json(updateResults);
     } else {
-      res.status(422).send(`No expression of interest with for user: ${userId}`);
+      res.status(422).send(`No expression of interest with for user`);
```
Comment (taraepp) [inline server/routes/participant-user.js:109]: Grammar: "No expression of interest found for user"

**Review by taraepp — CHANGES_REQUESTED** [review]
Just the grammar nitpick, otherwise looks good.

File: server/routes/participant-user.js (hunk 1 above)
Comment (alva-fresh) [reply server/routes/participant-user.js:109]: Actually, should we keep it consistent with _'No expression of interest found for this participant'_?

Comment (taraepp) [reply server/routes/participant-user.js:109]: Yeah, that's better.

**Review by alva-fresh — APPROVED** [review]

**Review by taraepp — APPROVED** [review]

**Pull request merged**